noir/noir-repo/aztec_macros/src/transforms/functions.rs

```diff
@@ -113,6 +113,90 @@ pub fn transform_function(
     Ok(())
 }
 
+pub fn export_fn_abi(
```

Comment:
Could you add here a comment about what the function does and what it is doing? Something like what you have in the PR desc would be enough:

"Leverages the #[abi(tag)] feature to output the aztec functions abi before macro transformation, so that values can be decoded ..."

Reply:
Done!